src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -28,16 +31,32 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             playerMsg(TFM_Command.PLAYER_NOT_FOUND);
             return true;
         }
+        else if (args.length > 1)
+        {
+            String reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+            smite(player, reason);
+            return true;
+        }
 
-        smite(player);
+        else
+        {
+            smite(player);
+        }
 
         return true;
     }
 
-    public static void smite(final Player player)
+    public static void smite(final Player player, final String reason)
     {
-        TFM_Util.bcastMsg(player.getName() + " has been a naughty, naughty boy.", ChatColor.RED);
+        if (reason == null)
```

Comment:
An if statement is an easier, and more efficient way to check for something. I think this should be kept.